bitnami/flux/values.yaml

```diff
@@ -218,6 +218,9 @@ kustomizeController:
   ## @param kustomizeController.args Override default container args (useful when using custom images)
   ##
   args: []
+  ## @param kustomizeController.automountServiceAccountToken Mount Service Account token in pod
+  ##
+  automountServiceAccountToken: true
```

Comment:
shouldn't it be set to `false` instead?

Reply:
They require access to the K8s API so they need to mount the SA token